src/Kinepolis/KinepolisPriceParser.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Kinepolis;
+
+final class KinepolisPriceParser implements PriceParser
+{
+    public function parseTheaterPrices(array $theaterPrices): ParsedPriceForATheater
+    {
+        $basePrice = 0;
+        $discountPrice = 0;
+        $studentPrice = 0;
+        $surchargeForLongMovie = 0;
+        $surchargeFor3D = 0;
+
+        // This is done because the array of prices of Kinepolis
+        // is always random and includes prices that are irrelevant for us
+        // like prices for various models of 3d glasses
+        // We assume the values below are always present
+        foreach ($theaterPrices as $theaterPrice) {
+            if ($theaterPrice[1] === 'Normaal tarief') {
```

Comment:
This entire structure could be a switch, maybe also move the strings to constants?